app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="d-flex flex-column justify-content-center p-4">
+<h1 class="h1 heading  align-self-center"><%=@recipe.name%></h1>
+<div  class="d-flex justify-content-between" >
+<p>Preparation time: <%=@recipe.preparationTime%> hours</p>
+
+<%= form_with model: @recipes do |form| %>
+ 
+     <%= form.check_box :public, { checked: @recipe.public, checked_value: true, unchecked_value: false }
+     %>
+    
+<% end %>
+</div>
+
```

Comment:
- [ ] According to the requirement this should be a toggle button, so that when you toggle it you can make the recipe public or private. Please make sure to implement the changes.